File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cur = 0
        ans = 0
        
        
        for i in range(len(nums)):
            if (nums[i] == 1):
                cur+=1
            elif (nums[i] == 0):
                ans = max(ans, cur)
                cur = 0
        
        return max(cur,ans)

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
       int cur = 0;
       int ans = 0;

       for (int i = 0; i < nums.length; i++) {
        if(nums[i] == 1){
            cur++;
        }
        else{
            ans = Math.max(ans, cur);
            cur = 0;
        }
       } 
       return Math.max(ans, cur);
    }
}